packages/@lwc/rollup-plugin/src/__tests__/fixtures/bundle/app.js

```diff
@@ -1,2 +0,0 @@
-import simpleClass from './simpleClass1.js';
-console.log(simpleClass);
```

Comment:
Following the pattern in @pmdartus 's previous [PR](https://github.com/salesforce/lwc/pull/2033), the tests are split into 4 categories: integrations, resolver, rootDir, compilerConfig for different testing purposes so many fixtures are moved around and some are added. 
Also these tests don't check against snapshots of the compiled js code but only check necessary assertions like no warnings from rollup so all snapshots of the expected compiled js files are removed. I assume PM did this change because checking against the entire compiled js code is not really necessary so I followed his PR.

Reply:
I've reverted all test changes in this PR and will put them in a separate PR as @pmdartus suggested.